parm/config/gfs/config.fcst

```diff
@@ -276,10 +277,10 @@ export deflate_level=1
 # Disable the use of coupler.res; get model start time from model_configure
 export USE_COUPLER_RES="NO"
 
-if [[ "$CDUMP" =~ "gdas" ]] ; then # GDAS cycle specific parameters
+if [[ "${CDUMP}" =~ "gdas" ]] ; then # GDAS cycle specific parameters
```

Comment:
Same things for these conditions. It should only keep gefs parts

Reply:
This is the GFS one.